packages/web-components/src/components/cbp-tooltip/cbp-tooltip.stories.tsx

```diff
@@ -0,0 +1,64 @@
+export default {
+    title: 'Components/Tooltip',
+    //tags: ['autodocs'],
+    argTypes: {
+        uid: {
+            description: 'A unique `id` applied to the dialog and referenced by the control.',
+            control: 'text',
+          },
+        open: {
+            description: 'toggle the open prop for the tooltip',
+            control: 'boolean'
+        },
+        alignment: {
+            description: 'Sets where the the tooltip will align to the control.',
+            control: 'select',
+            options: ["top-left", "top-center", "top-right", "right-top", "right-center", "right-bottom", "bottom-left", "bottom-center", "bottom-right", "left-top", "left-center", "left-bottom"]
+        },
+        context : {
+            control: 'select',
+            options: [ "light-inverts", "light-always", "dark-inverts", "dark-always"]
+        },
+        sx: {
+            description: 'Supports adding inline styles as an object of key-value pairs comprised of CSS properties and values. Values should reference design tokens when possible.',
+            control: 'object',
+        },
+    },
+  };
+
+const Template = ({ open, uid, alignment, title, content, buttonContent, context, sx }) => {
+return ` 
+        <a href='#' style='display: inline-block; margin-left:40%; margin-top:20%'> item #1</a>
+        <cbp-tooltip
+            ${open ? `open` : ''}
+            ${uid ? `uid=${uid}` : ''}
+            alignment=${alignment}
+            ${context && context != 'light-inverts' ? `context=${context}` : ''}
+            ${sx ? `sx=${JSON.stringify(sx)}` : ``}
+
+            >  
+            <div>
+                ${buttonContent}
+            </div>
+            <div slot="cbp-tooltip-content">
+                <div style='font-weight: var(--cbp-font-weight-bold)'>${title}</div>
```

Comment:
I feel like this text is too tight using divs with no line-height applied. I don't know if you can put a line height on the `role=tooltip` element and it inherits down or if you should use paragraph tags on them and remove bottom margin on the last child.